Assignment 3-JS/sass/style.scss

```diff
@@ -0,0 +1,235 @@
+      .month {
+          height: 20%;
+          display: flex;
+          justify-content: space-between;
+          align-items: center;
+          text-align: center;
+          padding: 0 20px;
+          border-bottom:2px solid #00e6b8;
+      }
+      .month i {
+          font-size: 20px;
+          cursor: pointer;
+          color: #00e6b8;
+      }
+      .month .date .month-name, .current-year-name {
+          text-transform: uppercase;
+          letter-spacing: 2px;
+          font-size: 30px;
+          color: #00e6b8;
+          cursor: pointer;
+      }
+      .week-days {
+          display: flex;
+          padding: 20px 0px;
+          
+      }
+      .week-days div {
+          width: calc(460px/8);
+          display: flex;
+          justify-content: center;
+          align-items: center;
+      }
+      .days-container {
+          display:flex;
+            #week-number{
+              .week {
+                  width:57.5px;
+                  height:60px;
+                  color:#34b7eb;
+                  display: flex;
+                  justify-content: center;
+                  align-items: center;
+                  box-sizing:border-box;
+              }
+          }
+      }
+      .days {
+          display: flex;
+          flex-wrap: wrap;
+          width:403.3px;
+      }
+      .days div {
+          width: calc(402.5px/7);
+          height: 60px;
+          display: flex;
+          justify-content: center;
+          align-items: center;
+          text-shadow: 2px 2px 5px rgb(48, 47, 47);
+      }
+      .prev-days,.next-days {
+          opacity: 0.3;
+      }
+      .sunday {
+          color: rgb(250, 100, 45);
+      }
+      .event {
+          border-radius:50%;
+          border: 2px solid red;
+          box-sizing:border-box;
+          color:red;
+      }
+      .today {
+          border:2px solid #00e6b8;
+          box-sizing: border-box;
+      }
+      .days div:hover:not(.week-number):not(.event){
+          box-sizing: border-box;
+          border: 2px solid white;
+      }
+  }
+  .sidepanel {
+      height: 0;
+      width: 460px; /* 0 width - change this with JavaScript */
+      position: fixed; /* Stay in place */
+      z-index: 1; /* Stay on top */
+      background-color: #00001a; /* Black*/
+      transition: 0.5s;
+      overflow: hidden;
+      box-shadow: 5px 5px 5px rgb(53, 53, 53);
+      text-align: center;
+      transform: scaleX(1);
+  }
+  /* The sidepanel links */
+  .sidepanel #years {
+      overflow: hidden;
+      scroll-behavior: smooth;
+      z-index: 1;
+      text-decoration: none;
+      font-size: 18px;
+      color: #818181;
+      transition: 0.3s;
+      cursor: pointer;
+      height: 115px;
+      display: flex;
+      flex-wrap: wrap;
+      align-items: center;
+      box-sizing: border-box;
+  }
+  .sidepanel #icon1 .next {
+      margin: 0;
+      padding: 0;
+  }
+  .sidepanel #icon1,#icon2 {
+      font-size:20px;
```

Comment:
Use it in a single place.

Reply:
Made a sass variable named $font-size and using it.